bridge/test/Federation_test.js

```diff
@@ -446,8 +408,14 @@ contract('FederationV2', async function (accounts) {
                 await this.allowTokens.setToken(originalTokenAddress, typeId);
                 this.sideTokenFactory = await SideTokenFactory.new();
                 this.bridge = await Bridge.new();
-                await this.bridge.methods['initialize(address,address,address,address)'](deployer, this.federators.address,
-                    this.allowTokens.address, this.sideTokenFactory.address);
+                await this.bridge.methods['initialize(address,address,address,address,string)']
```

Comment:
Why this signature changes if the only contract in the PR with changes is federation? was this test outdated?

Reply:
Same as before, they updated the contract but didn't updated the tests.